File: fzbz.js
function fizzbuzz(i) {
    let ret = ""
    if (i % 3 === 0) {
        ret += "fizz"
    }
    if (i % 5 === 0) {
        ret += "buzz"
    }
    if (ret === ""){
        ret = `${i}` 
    }
    return ret
}



for (let i = 0; i < 100; i++) {
    console.log(fizzbuzz(i));
}

File: a/fzbz.py
def fizzbuzz(i):
    ret = ""
    if not i % 3:
        ret += "fizz"
    if not i % 5:
        ret += "buzz"
    if not ret:
        ret = str(i)
    return ret

for i in range(100):
    print(fizzbuzz(i))
